aspnetcore/release-notes/aspnetcore-9.0.md

```diff
@@ -33,6 +35,6 @@ This section describes new features for authentication and authorization.
 
 ## Miscellaneous
 
-The following sections describe miscellaneous new features in ASP.NET Core 9.
+The following sections describe miscellaneous new features in ASP.NET Core 9.0.
```

Comment:
Grammar purists agreed and so do 95% of the editors. Personally I prefer the simpler 9. that has no ambiguity and is shorter, easier to read. We're never doing 9.1, so why waste the precious bits?

In your favor is consistency, so that alone justifies the change.

Reply:
Yes, I was just going by what we normally do ... "ASP.NET Core 8.0" but ".NET 8" otherwise. If we go one way or the other, then I'll update the Blazor docs to match.